tools/operator/paths/paths.go

```diff
@@ -158,13 +158,17 @@ func PTEPath() string {
 	return path
 }
 
-//GetConnProfilePathForOrg --
-func GetConnProfilePathForOrg(orgName string, organizations []inputStructs.Organization) string {
+//GetConnProfilePath --
+func GetConnProfilePath(orgNames []string, organizations []inputStructs.Organization) string {
 	var connProfilePath string
-	for i := 0; i < len(organizations); i++ {
-		if organizations[i].Name == orgName {
-			connProfilePath = organizations[i].ConnProfilePath
+	if len(orgNames) > 1 {
+		connProfilePath,_ = filepath.Split(organizations[0].ConnProfilePath)
```

Comment:
This establishes a requirement for the user:  if someone passes in more than one orgName, then all orgNames must use the same ConnProfilePath as specified in the first organizations[0]. Is that the intent? It enforces a restriction that each orgName cannot have its own ConnProfilePath. You should document this, where the user defines the orgNames.

Reply:
yes, @scottz64. Operator will anyway create and place connection profiles in the same directory. Expecting user to have all the connection profiles in the same directory if used to run against his custom network. Sure will document this assumption.